addon/helpers/queue.js

```diff
@@ -0,0 +1,23 @@
+// taken from Ember Composable Helpers (https://github.com/DockYard/ember-composable-helpers), then modified.
+import { helper } from '@ember/component/helper';
+import isPromise from '../utils/is-promise';
+
+export default helper(function queue(actions = []) {
+  return function (...args) {
+    let invokeWithArgs = function (acc, curr) {
+      if (isPromise(acc)) {
```

Comment:
I hate that this is here, random promise gets resolved in weird way. Just complaining, no need to change it in case we're using it somehwere.

Reply:
checked, and we're not feeding it any Promises anywhere. removed that feature from the  helper :heavy_check_mark: